workers/r_dependencies.R

```diff
@@ -0,0 +1,209 @@
+  "pd.clariom.s.mouse.ht",
+  "pd.clariom.s.rat",
+  "pd.clariom.s.rat.ht",
+  "pd.cotton",
+  "pd.cyngene.1.0.st",
+  "pd.cyngene.1.1.st",
+  "pd.cyrgene.1.0.st",
+  "pd.cyrgene.1.1.st",
+  "pd.cytogenetics.array",
+  "pd.drogene.1.0.st",
+  "pd.drogene.1.1.st",
+  "pd.drosgenome1",
+  "pd.drosophila.2",
+  "pd.ecoli",
+  "pd.e.coli.2",
+  "pd.ecoli.asv2",
+  "pd.elegene.1.0.st",
+  "pd.elegene.1.1.st",
+  "pd.equgene.1.0.st",
+  "pd.equgene.1.1.st",
+  "pd.feinberg.hg18.me.hx1",
+  "pd.feinberg.mm8.me.hx1",
+  "pd.felgene.1.0.st",
+  "pd.felgene.1.1.st",
+  "pd.fingene.1.0.st",
+  "pd.fingene.1.1.st",
+  "pd.genomewidesnp.5",
+  "pd.genomewidesnp.6",
+  "pd.guigene.1.0.st",
+  "pd.guigene.1.1.st",
+  "pd.hc.g110",
+  "pd.hg18.60mer.expr",
+  "pd.hg.focus",
+  "pd.hg.u133a",
+  "pd.hg.u133a.2",
+  "pd.hg.u133a.tag",
+  "pd.hg.u133b",
+  "pd.hg.u133.plus.2",
+  "pd.hg.u219",
+  "pd.hg.u95a",
+  "pd.hg.u95av2",
+  "pd.hg.u95b",
+  "pd.hg.u95c",
+  "pd.hg.u95d",
+  "pd.hg.u95e",
+  "pd.hta.2.0",
+  "pd.ht.hg.u133a",
+  "pd.ht.hg.u133.plus.pm",
+  "pd.ht.mg.430a",
+  "pd.hu6800",
+  "pd.huex.1.0.st.v2",
+  "pd.hugene.1.0.st.v1",
+  "pd.hugene.1.1.st.v1",
+  "pd.hugene.2.0.st",
+  "pd.hugene.2.1.st",
+  "pd.maize",
+  "pd.mapping250k.nsp",
+  "pd.mapping250k.sty",
+  "pd.mapping50k.hind240",
+  "pd.mapping50k.xba240",
+  "pd.margene.1.0.st",
+  "pd.margene.1.1.st",
+  "pd.medgene.1.0.st",
+  "pd.medgene.1.1.st",
+  "pd.medicago",
+  "pd.mg.u74a",
+  "pd.mg.u74av2",
+  "pd.mg.u74b",
+  "pd.mg.u74bv2",
+  "pd.mg.u74c",
+  "pd.mg.u74cv2",
+  "pd.mirna.1.0",
+  "pd.mirna.2.0",
+  "pd.mirna.3.0",
+  "pd.mirna.3.1",
+  "pd.mirna.4.0",
+  "pd.moe430a",
+  "pd.moe430b",
+  "pd.moex.1.0.st.v1",
+  "pd.mogene.1.0.st.v1",
+  "pd.mogene.1.1.st.v1",
+  "pd.mogene.2.0.st",
+  "pd.mogene.2.1.st",
+  "pd.mouse430.2",
+  "pd.mouse430a.2",
+  "pd.mta.1.0",
+  "pd.mu11ksuba",
+  "pd.mu11ksubb",
+  "pd.nugo.hs1a520180",
+  "pd.nugo.mm1a520177",
+  "pd.ovigene.1.0.st",
+  "pd.ovigene.1.1.st",
+  "pd.pae.g1a",
+  "pd.plasmodium.anopheles",
+  "pd.poplar",
+  "pd.porcine",
+  "pd.porgene.1.0.st",
+  "pd.porgene.1.1.st",
+  "pd.rabgene.1.0.st",
+  "pd.rabgene.1.1.st",
+  "pd.rae230a",
+  "pd.rae230b",
+  "pd.raex.1.0.st.v1",
+  "pd.ragene.1.0.st.v1",
+  "pd.ragene.1.1.st.v1",
+  "pd.ragene.2.0.st",
+  "pd.ragene.2.1.st",
+  "pd.rat230.2",
+  "pd.rcngene.1.0.st",
+  "pd.rcngene.1.1.st",
+  "pd.rg.u34a",
+  "pd.rg.u34b",
+  "pd.rg.u34c",
+  "pd.rhegene.1.0.st",
+  "pd.rhegene.1.1.st",
+  "pd.rhesus",
+  "pd.rice",
+  "pd.rjpgene.1.0.st",
+  "pd.rjpgene.1.1.st",
+  "pd.rn.u34",
+  "pd.rta.1.0",
+  "pd.rusgene.1.0.st",
+  "pd.rusgene.1.1.st",
+  "pd.s.aureus",
+  "pd.soybean",
+  "pd.soygene.1.0.st",
+  "pd.soygene.1.1.st",
+  "pd.sugar.cane",
+  "pd.tomato",
+  "pd.u133.x3p",
+  "pd.vitis.vinifera",
+  "pd.wheat",
+  "pd.xenopus.laevis",
+  "pd.x.laevis.2",
+  "pd.x.tropicalis",
+  "pd.yeast.2",
+  "pd.yg.s98",
+  "pd.zebgene.1.0.st",
+  "pd.zebgene.1.1.st",
+  "pd.zebrafish"
```

Comment:
I agree it is long and it is closer to data than code. However as far as I'm concerned this R file really is a configuration file that only runs during the build of the image. It just happens to be written in R. I'm not sure I see any benefit to moving this list of packages out of this file into another one, only to read it in as data.